frontend/components/compliance/ListOfCandidateResults.tsx

```diff
@@ -0,0 +1,155 @@
+import { useEffect, useState } from 'react';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import useTranslations from '../../hooks/useTranslation';
+import { getComplianceList } from '../../service/serviceAPI';
+import { CellValue, DataType } from '../../service/types';
+import Table from '../table/Table';
+import InfoModal from '../shared/modals/InfoModal';
+import Button from '../shared/buttons/Button';
+import { COMPLIANCE_TESTING_FORM } from '../../service/constants';
+import InfiniteScrollCustomLoader from '../InfiniteScrollLoader';
+import EvaluationSchemaTable from './EvaluationSchemaTable';
+
+const ListOfCandidateResults = () => {
+  const [resultData, setResultData] = useState<DataType>({ rows: [] });
+  const [currentDataLength, setCurrentDataLength] = useState(0);
+  const [allDataLength, setAllDataLength] = useState(0);
+  const [displayEvaluationSchemaModal, setDisplayEvaluationSchemaModal] =
+    useState(false);
+
+  const { format } = useTranslations();
+
+  const headers = [
+    'table.software_name.label',
+    'table.bb_specification.label',
+    'table.bb_version.label',
+    'test_table.status.label',
+    'table.submission_date.label',
+    'table.deployment_compliance.label',
+    'table.requirement_specification_compliance.label',
+    'table.interface_compliance.label',
+  ];
+
+  const fetchData = async (offset: number, limit: number) => {
+    const data = await getComplianceList(offset, limit);
+    if (data.status) {
+      setAllDataLength(data.data.count);
+
+      const transformedData: DataType = {
+        // @ts-ignore
+        rows: [...resultData.rows],
+      };
+
+      const propertyOrder = [
+        'softwareVersion',
+        'bb',
+        'bbVersion',
+        'status',
+        'submissionDate',
+        'deploymentCompliance',
+        'requirementSpecificationCompliance',
+        'interfaceCompliance',
+      ];
+
+      /*eslint no-prototype-builtins: */
+      for (const key in data.data.data) {
+        if (data.data.data.hasOwnProperty(key)) {
+          let subHeaderAdded = false;
+          transformedData.rows.push(
+            // @ts-ignore
+            ...data.data.data[key].map((item: object, index: number) => {
+              const cell: CellValue[] = [];
+              for (const property of propertyOrder) {
+                if (item.hasOwnProperty(property)) {
+                  if (property === 'status') {
+                    // @ts-ignore
+                    switch (item[property]) {
+                    case 0:
+                      cell.push({ value: 'Draft' });
```

Comment:
We currently only have english translations but to be consistent with other parts of the application I would recommend having it translated. This comment also applies to other cases of this switch.